### PR TITLE
Add Draco plugin

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: .github/actions/draco-install-dep/action.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    # Issues opened upstream:
+    # - https://github.com/google/draco/issues/954
+    # - https://github.com/google/draco/issues/955
+    - name: Patch Draco
+      if: steps.cache-draco.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}
+      shell: bash
+      run: |
+        cat $(pwd)/source/.github/actions/draco-install-dep/*.patch | patch -p1 -d $(pwd)/dependencies/draco
```
Comment (mwestphal) [inline .github/actions/draco-install-dep/action.yml:35]: single line, remove the | before

File: plugins/draco/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+find_package(draco REQUIRED)
+
+message(STATUS "Plugin: draco ${draco_VERSION} found")
+
+f3d_plugin_init()
+
+f3d_plugin_declare_reader(
+  NAME Draco
+  EXTENSIONS drc
+  MIMETYPES application/vnd.drc
```
Comment (mwestphal) [inline plugins/draco/CMakeLists.txt:12]: add to mime types

File: plugins/draco/module/vtkF3DDracoReader.cxx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+//----------------------------------------------------------------------------
+int vtkF3DDracoReader::RequestData(
+  vtkInformation*, vtkInformationVector**, vtkInformationVector* outputVector)
+{
+  vtkPolyData* output = vtkPolyData::GetData(outputVector);
+
+  std::vector<char> data;
+
+  auto reader = draco::StdioFileReader::Open(this->FileName);
+  if (!reader->ReadFileToBuffer(&data))
```
Comment (mwestphal) [inline plugins/draco/module/vtkF3DDracoReader.cxx:173]: can this throw ?

Comment (mwestphal) [reply plugins/draco/module/vtkF3DDracoReader.cxx:173]: generic question about draco

**Comment by mwestphal** [comment]
coverage not moving is suspect

File: plugins/draco/module/vtkF3DDracoReader.cxx (hunk 3 above)
Comment (Meakk) [reply plugins/draco/module/vtkF3DDracoReader.cxx:173]: It's not documented as such. It returns nullptr in case of failure.

**Comment by Meakk** [comment]
> coverage not moving is suspect

Indeed, I'll take a look

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -109,6 +114,7 @@ jobs:
         -DF3D_PLUGIN_BUILD_OCCT=ON
         -DF3D_PLUGIN_BUILD_ASSIMP=ON
         -DF3D_PLUGIN_BUILD_ALEMBIC=ON
+        -DF3D_PLUGIN_BUILD_DRACO=ON
```
Comment (mwestphal) [inline .github/workflows/ci.yml:117]: sort alphabetically

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -303,12 +313,14 @@ jobs:
...
```
Comment (mwestphal) [inline .github/workflows/ci.yml:316]: sort

**Comment by Meakk** [comment]
Replaced by https://github.com/f3d-app/f3d/pull/526

**Pull request closed without merging**